### PR TITLE
76 featchannel mute ban

### PR DESCRIPTION
**Pull request by Tywael**

### CODE REVIEW

**Review by MarJC5 — COMMENTED** [review]
- When user is muted, he's still able to sent message (must check with @ldominiq to use his function to detect if user is muted, should be the same as blocked)
- When user is ban, he's still in the channel room and can still sent message (must be redirected to channels lobby router.push({name: "channels"}) )

File: project/src/web/app/src/components/chats/ChatConversation.vue [hunk 1]
```diff
@@ -57,22 +57,49 @@ export default defineComponent({
...
+              user: userStore.user,
+              chatId: props.chat.id || route.params.id,
+            };
+            props.socket.emit("newMessage", message);
+            newMessage.value = "";
+          } 
+        }
+      })
+      .catch((error) => {
+        console.log(error);
```
Comment (ldominiq) [inline project/src/web/app/src/components/chats/ChatConversation.vue:95]: NON NON NON VIREZ MOI CA xD

**Review by ldominiq — DISMISSED** [review]
Mute et kick OK ! Ban fonctionnel a moitie, faut que je sorte du channel sinon je peux toujours envoyer des messages

**Review by MarJC5 — APPROVED** [review]
LGTM

**Pull request merged**